runtime/jvmti/jvmtiHelpers.c

```diff
@@ -772,6 +772,89 @@ getThreadState(J9VMThread *currentThread, j9object_t threadObject)
 	return state;
 }
 
+#if JAVA_SPEC_VERSION >= 19
+jint
+getVirtualThreadState(J9VMThread *currentThread, jthread thread, BOOLEAN hasVMAccess)
```

Comment:
I think the caller must always have VM access

Reply:
we should assert this

Reply:
Fixed.